go.mod

```diff
@@ -71,6 +71,8 @@ require (
 )
 
 replace (
+	github.com/cortexproject/cortex => github.com/pracucci/cortex v0.0.0-20210222171122-5c74b43721b8
```

Comment:
To temporarily fix the Cortex compilation issue because of the changed `Iter()` signature. The little change is here:
https://github.com/thanos-io/thanos/compare/pracucci:add-list-recursively

I will open a follow up PR to remove this and upgrade Cortex once this PR will be merged (I will need to upgrade Thanos in Cortex, fix it in Cortex and then remove this override).